x-pack/plugins/infra/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -218,11 +217,7 @@ export class KibanaFramework {
   }
 
   public getIndexPatternsService(requestContext: RequestHandlerContext): IndexPatternsFetcher {
-    return new IndexPatternsFetcher((...rest: Parameters<LegacyAPICaller>) => {
-      rest[1] = rest[1] || {};
-      rest[1].allowNoIndices = true;
-      return requestContext.core.elasticsearch.legacy.client.callAsCurrentUser(...rest);
-    });
+    return new IndexPatternsFetcher(requestContext.core.elasticsearch.client.asCurrentUser, true);
```

Comment:
What is `true` signifying in this signature? (I assume it's the `allowNoIndices` flag based on what was there before but just checking?)

Reply:
correct!

Reply:
Thanks! I think my least favorite thing about TS is that it let us all go back to these dangling signatures :P When does GitHub get full TS support??? ;)